circuits/integration/zk_circuits/valid_match_mpc.rs

```diff
@@ -127,6 +127,8 @@ async fn match_orders(
         base_amount: min_base_amount,
         quote_amount,
         direction: party0_order.side.into(),
+        protocol_base_fee_amount: 0,
+        protocol_quote_fee_amount: 0,
```

Comment:
Dummy values, planning to update when the constraints are written (tests will fail if not updated then)